sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -232,6 +242,99 @@ export class AssessmentController {
     }
   }
 
+  /**
+   * Verify as per the order of original assessment study start date if this assessment
```

Comment:
Maybe it would be worth to mentioned that assessments pending PIR would not be considered. Looking at the query and getting it from `assessment.offering IS NOT NULL` may be tricky.